gdraw/gscrollbar.c

```diff
@@ -520,8 +520,7 @@ static void GScrollBarFit(GScrollBar *gsb) {
     minheight = 2*(gsb->thumbborder+gsb->arrowsize) + GDrawPointsToPixels(gsb->g.base,2);
 
     if ( gsb->g.vert ) {
-	if ( gsb->g.r.width==0 )
-	    gsb->g.r.width = GDrawPointsToPixels(gsb->g.base,_GScrollBar_Width);
+	gsb->g.r.width = GDrawPointsToPixels(gsb->g.base,_GScrollBar_Width);
```

Comment:
The commit message says it itself: it's about having scrollbars in the lookup pane with the same width as the other ones in the program.